Indico/Mutation/DocumentExtraction.cs

```diff
@@ -8,22 +8,36 @@
 
 namespace Indico.Mutation
 {
+    /// <summary>
+    /// OCR PDF, TIF, JPG and PNG files
+    /// </summary>
     public class DocumentExtraction : Mutation<List<Job>>
     {
         List<string> _files;
         JObject _jsonConfig;
         IndicoClient _client;
 
+
+        public JObject JsonConfig

```

Comment:
C# has default getters and setter. This can be written as JObject JsonConfig {get,set}